common/bitvec/src/lib.rs

```diff
@@ -31,15 +31,14 @@ const MAX_BUCKETS: usize = 32;
 /// * A position can set more than once -- it remains set after the first time.
 ///
 /// # Examples:
-/// ```ignore
 /// let mut bv = BitVec::default();
 /// bv.set(2);
 /// bv.set(5);
 /// assert!(bv.is_set(2));
 /// assert!(bv.is_set(5));
 /// assert_eq!(false, bv.is_set(0));
-/// assert_eq!(bv1.count_ones(), 2);
-/// assert_eq!(bv1.last_set_bit(), 3);
+/// assert_eq!(bv.count_ones(), 2);
+/// assert_eq!(bv.last_set_bit(), 5);
```

Comment:
It look like `last_set_bit` returns an `Option<_>`, so you want

```suggestion
/// assert_eq!(bv.last_set_bit(), Some(5));
```